Tools/CISupport/ews-build/steps.py

```diff
@@ -3130,7 +3130,9 @@ class CompileWebKit(shell.Compile, AddToLogMixin):
     env = {'MFLAGS': ''}
     warningPattern = '.*arning: .*'
     haltOnFailure = False
-    command = ['perl', 'Tools/Scripts/build-webkit', WithProperties('--%(configuration)s')]
+    build_command = ['perl', 'Tools/Scripts/build-webkit']
+    filter_command = ['perl', 'Tools/Scripts/filter-build-webkit', '-logfile', 'WebKitBuild/build-log.txt']
+    command = ['/bin/sh', '-c']
```

Comment:
Do we actually need to set this command here? it seems weird/convoluted to set half of the command here (which gives no indication of what it will do), and then complete it later. Might as well just set it once later-on.